samples-final/AspNetCore-RichCards-Bot/RichCardsBot.cs

```diff
@@ -0,0 +1,549 @@
+            var activity = dialogContext.Context.Activity;
+            switch (selectedCard)
+            {
+                case "Adaptive card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateAdaptiveCardAttachment()));
+                    break;
+                case "Animation card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateAnimationCardAttachment()));
+                    break;
+                case "Audio card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateAudioCardAttachment()));
+                    break;
+                case "Hero card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateHeroCardAttachment()));
+                    break;
+                case "Receipt card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateReceiptCardAttachment()));
+                    break;
+                case "Signin card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateSignInCardAttachment()));
+                    break;
+                case "Thumbnail card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateThumbnailCardAttachment()));
+                    break;
+                case "Video card":
+                    await dialogContext.Context.SendActivity(CreateResponse(activity, CreateVideoCardAttacment()));
+                    break;
+                default: // all cards
+                    await dialogContext.Context.SendActivities(new Activity[]
+                    {
+                        CreateResponse(activity, CreateAdaptiveCardAttachment()),
+                        CreateResponse(activity, CreateAnimationCardAttachment()),
+                        CreateResponse(activity, CreateAudioCardAttachment()),
+                        CreateResponse(activity, CreateHeroCardAttachment()),
+                        CreateResponse(activity, CreateReceiptCardAttachment()),
+                        CreateResponse(activity, CreateSignInCardAttachment()),
+                        CreateResponse(activity, CreateThumbnailCardAttachment()),
+                        CreateResponse(activity, CreateVideoCardAttacment())
+                    });
+                    break;
+            }
+            await dialogContext.End();
+        }
+
+        private Activity CreateResponse(Activity activity, Attachment attachment)
+        {
+            var response = activity.CreateReply();
+            response.Attachments = new List<Attachment>() { attachment };
+            return response;
+        }
+
+        #region AdaptiveCardRaw
+        private const string AdaptiveCardRaw = @"
+{
+  ""$schema"": ""http://adaptivecards.io/schemas/adaptive-card.json"",
```

Comment:
Consider putting this JSON into a file, and just reading it in. I think that would be much similar, as the esacping goes away, and the size of the file goes away.